docker-image-src/4.0/docker-entrypoint.sh

```diff
@@ -403,21 +403,12 @@ if [ "${cmd}" == "neo4j" ]; then
 fi
 
 declare -A COMMUNITY
-declare -A ENTERPRISE
+# declare -A ENTERPRISE
 
 COMMUNITY=(
      [dbms.tx_log.rotation.retention_policy]="100M size"
      [dbms.memory.pagecache.size]="512M"
      [dbms.default_listen_address]="0.0.0.0"
-     [dbms.connector.https.advertised_address]="0.0.0.0:7473"
-     [dbms.connector.http.advertised_address]="0.0.0.0:7474"
-     [dbms.connector.bolt.advertised_address]="0.0.0.0:7687"
-)
-
-ENTERPRISE=(
```

Comment:
How do you feel about keeping the Enterprise array, even if its empty, just to indicate to future authors *how* you should add Enterprise overrides?